xds/internal/resolver/xds_resolver_test.go

```diff
@@ -2024,18 +1479,21 @@ func (s) TestXDSResolverHTTPFilters(t *testing.T) {
 					res.OnCommitted()
 					doneFunc()
 
+					gotPaths := fb.paths
```

Comment:
I think I may have misspoken.

So, earlier there was a pointer to a slice where the paths were recorded. I changed it so that we could simply use a slice instead of a pointer to a slice. I did explore using a channel, but that didn't work out so well (which is why you see some remnants of that approach). But hopefully I have cleaned it up completely this time around.

There is a `cmp.Equal(gotPath, remainingWant[i])`. This is the verification.

Reply:
Ah ok yeah you mentioned using both slice and channel. Just using slice sgtm.